compiler/damlc/tests/daml-test-files/Interface.EXPECTED.md

```diff
@@ -14,30 +14,58 @@
 >   
 >   (no fields)
 
-> * **implements** Token
+> * **implements** [Token](#type-interface-token-72202)
+
+## Interfaces
```

Comment:
This does currently not work as far as I can tell, since we ignore the doc strings in the parser. For the choices it should work :tm: . I will check and change this in the next PR.